kratos/processes/find_nodal_neighbours_process.h

```diff
@@ -79,11 +74,20 @@ class FindNodalNeighboursProcess
     /// avg_elems ------ expected number of neighbour elements per node.,
     /// avg_nodes ------ expected number of neighbour Nodes
     /// the better the guess for the quantities above the less memory occupied and the fastest the algorithm
-    FindNodalNeighboursProcess(ModelPart& model_part, unsigned int avg_elems = 10, unsigned int avg_nodes = 10)
-        : mr_model_part(model_part)
+    FindNodalNeighboursProcess(ModelPart& model_part,
+        const DataCommunicator& rComm = ParallelEnvironment::GetDefaultDataCommunicator(),
+        bool compute_neighbour_nodes = true,
+        bool compute_neighbour_elements = true
+    )
```

Comment:
A Flags would be better as it would be more clear when it is used

Reply:
before i do any change here, we should decide if we do want to offer this possiblity.

In a sense i would love to remove the "FindNodalNeighbour" process ... 

one would have to call either the one to compute the NEIGHBOUR_NODES or the one for NEIGHBOUR_ELEMENTS. This way one would only call the one that he actually needs.

Reply:
I agree with this objective.  
I thought you use the flags for when you need both but checking the implementation you already call them separately. By the way a local flag wouldn't harm here.

Reply:
the point is to avoid work for nothing.
 if we decide not to offer this option (which i am currently inclined towards) i would simply remove it without implementing the flags stuff.

Reply:
So lets put the legacy one calculating both as before and we can substitute to new ones when we want to calculate one of them.

Reply:
meaning to remove the options from the constructor? (ok for me)

Reply:
I would go for removing the options if you also agree!

Reply:
options removed